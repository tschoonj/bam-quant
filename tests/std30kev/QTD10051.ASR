File: C:\xrfdaten\2014KW30\chris\std30kev\std10051.spe
Start: 22.07.2014 10:22:20
Stop: 22.07.2014 10:24:00
BESSY: 298.1642415 298.1924878
IONI: 1.945786E-11 1.962377E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 70157
Run: 100000
Dead: 30
ICR: 97960
OCR: 0
Accumulated BESSY: 59693.0991768
Accumulated Ioni: 3.90466442E-09
N Accumulation: 200
Normfactor:  7.3008810e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Sb
exp_x 37.7203979492188 37.7204
exp_y 150.465789794922 150.4658
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  51   1     26.272   5146726.      2268.     110.72
