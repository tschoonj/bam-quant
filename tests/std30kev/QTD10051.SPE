$DATA:
0 2000
15850
10766
5251
1949
601
194
67
36
13
10
4
4
4
6
10
28
310
3620
5291
5340
5038
4981
4907
4700
4719
4600
4552
4603
4414
4262
4251
4101
4089
4107
4033
4016
3761
3844
3742
3834
3578
3569
3556
3432
3507
3401
3430
3317
3382
3308
3313
3275
3247
3271
3217
3169
3052
3104
3100
3092
2989
3029
3198
2963
3021
3048
2968
3023
2963
2939
3052
3038
3184
3138
3155
3104
3086
3168
3133
3178
2965
3105
3067
3097
3121
3032
3182
3119
3325
3266
3259
3267
3387
3432
3490
3530
3487
3476
3600
3685
3739
3775
3772
3860
3729
3749
3583
3740
3614
3580
3511
3340
3394
3346
3198
3200
3123
3145
2998
3024
2886
2804
2724
2678
2621
2546
2436
2317
2289
2199
2127
2058
2060
1982
1840
1858
1866
1776
1816
1705
1638
1658
1584
1610
1585
1524
1482
1382
1492
1364
1339
1341
1235
1238
1216
1211
1181
1102
1074
1028
1035
997
936
924
952
923
895
877
882
801
827
870
774
742
701
759
761
746
651
658
656
692
661
625
677
640
608
612
590
599
518
563
603
564
545
559
554
521
517
514
470
532
438
499
464
470
510
452
461
439
428
460
443
374
448
445
439
445
390
407
416
411
420
434
397
380
390
420
383
387
430
423
385
376
390
360
369
354
379
362
369
363
347
330
323
329
340
363
344
352
347
333
334
313
345
310
324
320
344
311
316
329
325
316
335
399
450
423
461
467
446
398
341
321
310
269
343
306
291
311
297
333
292
274
280
286
279
288
310
311
272
308
302
326
301
311
300
300
296
282
278
282
294
279
260
274
274
261
277
284
323
352
406
499
577
717
799
801
730
663
600
522
387
315
298
264
312
258
245
265
300
268
268
256
264
258
247
270
224
239
253
252
236
314
288
295
289
308
308
336
316
328
277
293
278
245
238
255
234
247
222
226
202
227
208
257
287
281
309
286
273
309
285
274
267
271
266
246
243
231
230
201
215
249
279
224
250
240
248
238
217
227
220
233
252
261
243
263
278
253
252
251
224
227
219
220
229
244
234
213
212
194
249
225
199
226
256
231
236
230
213
220
189
232
229
208
231
209
239
216
225
206
233
203
219
230
200
213
219
240
231
230
228
245
212
221
235
219
222
227
232
231
227
219
250
255
232
273
231
255
279
275
223
192
232
209
198
214
220
200
212
216
207
223
207
216
221
206
205
185
208
212
206
219
197
231
203
193
195
190
208
204
206
198
215
226
197
215
195
200
197
226
189
192
198
188
193
163
214
223
222
198
188
197
208
205
206
210
203
204
204
196
195
213
184
219
214
221
200
199
191
197
191
195
236
221
186
229
189
214
194
189
210
186
214
208
188
196
196
167
191
185
187
218
210
205
196
208
205
193
193
166
194
179
205
182
209
196
182
199
183
181
164
185
195
190
190
182
195
180
175
190
166
209
197
208
212
166
178
162
187
177
174
176
187
193
194
180
169
162
167
204
206
202
190
175
192
187
182
180
193
200
202
204
183
178
188
191
187
188
197
215
228
220
239
198
223
226
204
225
206
241
209
205
189
172
169
209
192
211
212
212
225
218
235
226
213
226
215
178
220
215
224
189
197
191
216
203
181
179
189
181
213
184
178
205
191
182
172
166
192
181
174
195
171
181
189
168
179
172
169
158
172
188
179
182
195
168
224
168
181
210
214
186
180
218
193
211
172
198
176
182
185
191
162
204
192
194
193
197
234
199
194
181
184
189
185
221
214
169
209
208
192
199
226
223
237
237
219
210
242
258
254
269
271
261
247
222
255
251
247
261
262
258
290
287
293
305
277
254
281
288
289
280
270
271
302
307
295
298
336
303
358
408
431
535
691
1028
1427
2097
3141
4203
6037
7943
10263
12830
15123
17072
18520
19848
20521
19630
18312
16876
14045
11960
9328
6947
5019
3641
2310
1556
1045
723
514
386
312
271
248
246
238
219
203
187
232
210
216
166
196
195
192
209
187
183
166
191
195
170
187
175
174
178
191
208
177
157
201
213
209
190
185
185
184
179
196
183
195
176
206
190
205
182
203
172
199
219
215
192
203
205
203
212
212
204
220
218
195
270
255
269
288
331
393
443
505
631
883
1097
1417
1805
2223
2548
2880
3234
3323
3303
3268
2902
2652
2254
1876
1566
1215
992
849
745
685
625
711
648
663
635
554
545
411
397
329
310
253
234
200
179
188
141
184
173
157
161
150
156
152
176
153
160
195
172
171
145
162
163
151
176
181
165
141
162
159
158
143
166
151
145
165
150
156
177
154
170
181
150
143
163
165
162
168
140
155
167
171
161
159
172
175
172
167
176
156
164
180
129
175
163
143
178
192
161
184
156
171
184
193
188
190
198
211
161
177
167
148
162
170
147
178
188
181
197
146
186
169
187
187
167
187
173
170
188
155
199
196
201
191
167
198
190
192
196
197
203
160
189
196
186
173
202
190
207
220
175
178
196
195
191
179
224
184
223
198
208
206
215
198
209
202
215
217
215
211
222
226
233
208
227
214
223
209
223
211
222
245
214
235
210
228
208
208
223
237
256
244
268
233
222
260
252
270
226
278
253
237
220
252
284
260
272
236
279
286
260
276
244
262
262
266
270
252
287
321
250
303
278
272
316
266
262
290
304
268
278
284
308
282
279
305
311
310
294
315
314
307
308
290
321
324
309
324
326
363
307
365
342
389
381
334
367
387
390
364
368
384
404
396
415
412
439
442
433
472
441
427
443
479
504
483
480
482
497
510
506
524
574
574
562
580
649
609
629
644
624
731
741
684
765
728
791
821
826
924
907
948
970
941
1005
1049
1074
1125
1090
1159
1188
1184
1210
1252
1298
1376
1348
1323
1332
1430
1391
1417
1558
1500
1497
1610
1501
1538
1548
1595
1682
1636
1606
1613
1668
1620
1698
1617
1721
1664
1681
1646
1647
1620
1630
1695
1662
1629
1676
1680
1659
1662
1612
1651
1609
1626
1463
1535
1445
1483
1488
1398
1439
1413
1413
1389
1467
1435
1444
1522
1548
1518
1497
1694
1688
1722
1779
1735
1738
1882
1805
1842
1996
1984
1936
1961
2010
2066
2081
1936
1934
1990
1940
1978
1899
1997
2077
2017
2075
2126
2297
2345
2436
2440
2685
2837
3074
3421
4096
4749
5945
7436
9992
13505
17840
24159
31818
41590
53609
65832
80144
94448
109807
122269
133498
145121
153287
159939
166847
172226
180110
189430
199959
212006
222824
234718
242048
244815
243828
235887
222938
204067
181905
157645
131495
107860
85402
66018
49568
36592
26001
18535
13061
9229
6588
4851
3723
2903
2358
2057
1832
1605
1473
1333
1213
1197
1236
1095
1062
1049
1019
971
913
879
832
837
804
798
729
689
756
665
686
630
641
647
619
607
579
575
625
602
534
573
526
556
560
538
529
494
491
478
498
511
472
481
494
500
493
482
471
441
496
464
483
476
488
507
472
491
473
442
490
418
442
486
467
457
464
414
496
466
486
471
476
506
483
525
544
540
584
587
559
622
610
556
561
582
569
566
544
520
554
550
546
541
514
543
519
526
572
526
564
571
560
600
586
571
590
566
601
636
621
660
617
665
686
693
676
776
805
851
849
989
1062
1254
1570
1974
2579
3354
4337
6064
8110
10452
13760
17159
21603
25960
31039
35387
40053
43774
46577
48422
48567
48111
45846
42887
39294
34283
29615
24677
20318
16495
12714
9842
7565
5717
4277
3315
2537
2134
1845
1857
1896
2112
2405
2921
3541
4194
5093
5944
6839
7827
8407
9085
9364
9532
9297
9165
8572
7930
6957
6191
5062
4248
3342
2815
2114
1637
1154
818
635
453
312
205
169
123
114
83
68
69
60
52
49
43
62
51
42
43
41
34
43
52
41
39
49
43
36
28
44
32
37
28
23
30
27
31
35
38
37
45
27
44
40
33
38
29
32
34
34
33
33
29
24
32
24
28
43
37
27
32
34
31
30
24
19
26
17
14
21
25
16
28
17
24
19
20
29
18
17
20
17
21
22
25
14
18
24
12
21
17
25
23
19
18
19
23
19
14
31
18
20
18
25
21
18
19
28
13
17
21
20
15
19
16
18
15
20
6
14
10
18
14
13
14
15
12
13
13
13
13
21
13
15
5
13
12
15
9
14
11
18
10
13
15
13
13
7
18
17
8
12
8
16
16
7
12
15
15
7
10
10
16
11
12
15
17
18
17
15
11
18
16
18
14
16
18
16
12
12
13
15
14
16
16
12
9
18
9
16
16
12
16
10
17
17
15
15
15
16
16
9
12
16
18
15
16
17
24
17
16
27
20
25
25
23
18
15
20
26
25
30
14
23
18
20
19
24
17
19
21
13
22
12
10
16
7
14
13
9
13
11
9
12
8
11
11
8
6
6
5
8
4
4
5
2
3
6
3
5
5
6
10
5
3
7
7
5
3
7
6
4
7
4
3
6
5
6
3
6
3
4
9
3
10
4
3
4
2
6
2
4
8
2
2
4
2
4
9
5
6
9
7
2
3
3
1
6
4
3
5
5
2
2
4
5
3
0
5
3
6
5
4
8
5
4
4
3
8
5
4
2
6
4
2
3
5
4
5
3
3
1
2
6
3
4
6
3
5
3
3
3
1
4
8
1
3
2
4
5
7
3
2
3
4
3
4
2
6
4
4
3
2
2
3
4
5
4
4
2
9
3
2
4
4
4
8
2
6
8
7
7
8
4
4
4
2
4
4
7
2
3
3
1
1
5
1
1
5
5
2
7
4
2
5
9
2
0
2
8
9
5
1
4
2
3
2
5
6
7
6
5
4
3
7
5
6
7
4
2
4
6
4
4
2
3
4
8
6
5
6
3
6
6
2
4
7
7
4
4
2
9
3
5
6
6
4
5
3
7
8
5
4
3
3
7
2
2
9
4
6
6
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10051.spe
Start: 22.07.2014 10:22:20
Stop: 22.07.2014 10:24:00
BESSY: 298.1642415 298.1924878
IONI: 1.945786E-11 1.962377E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 70157
Run: 100000
Dead: 30
ICR: 97960
OCR: 0
Accumulated BESSY: 59693.0991768
Accumulated Ioni: 3.90466442E-09
N Accumulation: 200
Normfactor:  7.3008810e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Sb
 
exp_x 37.7203979492188 37.7204
exp_y 150.465789794922 150.4658
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  51   1     26.272   5146726.      2268.     110.72
