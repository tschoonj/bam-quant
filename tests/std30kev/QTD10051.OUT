     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:57  บ   
     บ  Spectrum: QTD10051.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        2 iterations done    บ   
     บ  ChiSquare =   367.4   last change =   .06%    lambda= 1.E-04   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -14.4 ๑   100.0   บ     -14.4 ๑      .9   บ   
     บ  GAIN (eV/ch)  บ     20.197 ๑   2.000   บ    20.197 ๑    .001   บ   
     บ det NOISE (eV) บ      122.8 ๑    40.0   บ     122.8 ๑     5.6   บ   
     บ  FANO factor   บ       .118 ๑    .050   บ      .118 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Sb-Ka                           5146726. ๑   2268.           บ   
     บ    KA1        26.359    .64820     3336123. ๑   1471.    132.6  บ   
     บ             1305.840    285.19        1633.           7.56E-01  บ   
     บ    KA2        26.111    .35180     1810602. ๑    799.     72.0  บ   
     บ             1293.561    284.10        1686.           7.65E-01  บ   
     บ                                                                 บ   
     บ  2 Sb-Kb                            911485. ๑    956.           บ   
     บ    KB1        29.726    .55613      506901. ๑    532.     39.5  บ   
     บ             1472.552    299.66        1025.           6.38E-01  บ   
     บ    KB3        29.679    .28683      261444. ๑    276.     40.4  บ   
     บ             1470.225    299.46        1033.           6.39E-01  บ   
     บ    KB2        30.390    .15704      143139. ๑    153.     45.7  บ   
     บ             1505.429    302.43         895.           6.16E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
